var/spack/repos/builtin/packages/r-bibtex/package.py

```diff
@@ -11,9 +11,9 @@ class RBibtex(RPackage):
 
        Utility to parse a bibtex file."""
 
-    homepage = "https://cran.r-project.org/package=bibtex"
-    url      = "https://cran.r-project.org/src/contrib/bibtex_0.4.2.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/bibtex/"
+    homepage = "https://cloud.r-project.org/package=bibtex"
+    url      = "https://cloud.r-project.org/src/contrib/bibtex_0.4.2.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/bibtex/"
 
     version('0.4.2', sha256='1f06ab3660c940405230ad16ff6e4ba38d4418a59cd9b16d78a4349f8b488372')
 
```

Comment:
The `r-testthat` dependency is suggested, not required. It should be removed.

Reply:
Done.